modules/docs/arrow-docs/docs/_sass/components/_navigation.scss

```diff
@@ -41,6 +41,59 @@
               }
             }
         }
+
+        .dropbtn {
+          @include links($white, $white, rgba($white, 0.5), rgba($white, 0.5));
+
+          cursor: pointer;
+          border: none;
+          background: transparent;
+          color: $white;
+          font-family: $brand-font-family;
```

Comment:
This ends up resolving to “Source Sans Pro”, when the rest of the items in the top menu are using “Titillium Web”. We need some consistency there.

Reply:
Ok, I will remove that line.

Reply:
:thinking: doesn't look like you did it in the end?